rules.ini

```diff
@@ -47,6 +47,7 @@ CryEngine[] = (?:^|/)cry3dengine\.dll$
 CryEngine[] = (?:^|/)CryD3DCompilerStub\.dll$
 CryEngine[] = (?:^|/)CryRenderD3D1[12]\.dll$
 CryEngine[] = (?:^|/)CryRenderVulkan\.dll$
+Danmakufu = th_dnh\.def$
```

Comment:
Add `(?:^|/)` otherwise `abcth_dnh.def` matches (which should be added to non matching tests).